server/static/dashboard/collab.js

```diff
@@ -7,42 +10,486 @@ collab.fetch(function(err) {
   document.querySelector('#partners').textContent = collab.data.users.slice().sort().join(' & ')
 });
 
+// Parameters required to use updateFunction
+// TODO: Reconcile parameters and extraArgs
+var parameters = {};
```

Comment:
Make local. `extraArgs` → rename to... what? Or use a different mechanism to pass around the necessary data.

Reply:
Basically removed.